docs-src/zdm-core/modules/migrate/pages/introduction.adoc

```diff
@@ -47,46 +47,77 @@ For additional terms, see the xref:glossary.adoc[glossary].
 An important migration prerequisite is that you already have the matching schema on Target. A CQL statement that your client application sends to {zdm-proxy} must be able to succeed on both Origin and Target clusters. This means that any keyspace that your client application uses must exist on both Origin and Target with the same name. Table names must also match. For more, see xref:feasibility-checklists.adoc#_schemakeyspace_compatibility[Schema/keyspace compatibility].
 ====
 
-Your migration project occurs through a sequence of phases, which matches the structure of this ZDM documentation.
+Here's an overview of the migration phases, which matches the structure of this ZDM documentation.
 
 image::{imagesprefix}migration-all-phases.png[Migration phases from start to finish.]
```

Comment:
Why not replace this with the interactive graphic? It is redundant to have them in back-to-back sections.

Reply:
Thanks @jgillenwater . Funny, @alicel  and I were just chatting about this very thing!  I proposed keeping both, to over-communicate what's about to happen in each phase, for users who are brand new to ZDM and easing into their understanding.  But if the redundancy occurred to you both, maybe I should remove the purple version.  Ok.  I'll comment out the first one.  Will need to tweak some text too.   :-)

Reply:
Done in most recent commit.  Will rebuild coppi version in a few minutes...